docs/source/index.mdx

```diff
@@ -121,6 +122,7 @@ type Review {
 
 extend type User @key(fields: "id") {
   id: ID! @external
+  username: String! @external
```

Comment:
This is inappropriately marked as external. Only fields that are referenced by a key, requires, or provides should be marked external.
https://github.com/apollographql/federation/blob/c3bd6ddca9ed148c71eddb7969a5030148f3b582/docs/source/errors.md#L35

Reply:
Ah, so, `username` _is_ used in a `@provides`, but up in the `Review` type just above here. If I try to compose these exact subgraphs w/ Rover and _don't_ include `username` here with `@external`, I get a composition error.

Reply:
Ohhh, then great catch! At the glance I took, I wasn't even aware of the review type / that the SDL was collapsed 👍